src/serialbox-python/pp_ser/pp_ser.py

```diff
@@ -1,6 +1,13 @@
 #!/usr/bin/env python
-# This file is released under terms of BSD license`
-# See LICENSE.txt for more information
+# -*- coding: utf-8 -*-
```

Comment:
Was this header really necessary?

Reply:
Well I would aim for consistency.. either remove all the headers if you don't like it or put atleast the same on every file -- I don't care which way you want to go ;)

Reply:
I was just teasing your ASCII art skills